#!/usr/bin/env python
PACKAGE = "omnidirectional_robot_odometry"
NODE = "compute_odometry"
NAME = "parameters"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

integration_enum = gen.enum([   gen.const("Euler", int_t, 0, "Euler integration"),
                                gen.const("RK", int_t, 1, "Runge Kutta integration")],
                                "Enum of odometry integrations")
gen.add("integration_method", int_t, 0, "Format of count", 0, 0, 1, edit_method=integration_enum)


exit(gen.generate(PACKAGE, NODE, NAME))
